crates/nostr/src/event/builder.rs

```diff
@@ -1706,6 +1715,61 @@ impl EventBuilder {
     pub fn poll_response(response: PollResponse) -> Self {
         response.to_event_builder()
     }
+
+    /// Chat message
+    ///
+    /// <https://github.com/nostr-protocol/nips/blob/master/C7.md>
+    #[inline]
+    pub fn chat_message<S>(content: S) -> Self
+    where
+        S: Into<String>,
+    {
+        Self::new(Kind::ChatMessage, content)
+    }
+
+    /// Chat message reply
+    ///
+    /// <https://github.com/nostr-protocol/nips/blob/master/C7.md>
+    #[inline]
```

Comment:
@yukibtc Do you think the `#[inline]` should be removed? Because it's not a small function anymore